pom.xml

```diff
@@ -29,6 +29,10 @@
     <tag>HEAD</tag>
   </scm>
 
+  <properties>
+    <dep.kafka.version>0.10.1.0</dep.kafka.version>
+  </properties>
+
```

Comment:
That would be pretty bad.  Do we not know which clusters are still on 9?

cc @SegFaultAX